spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -88,11 +88,11 @@
     context 'opt in false at start but true during navigation' do
       let(:initial_opt_in_enabled) { false }
 
-      it 'should not be bounced back from Hybrid Handoff to How to Verify' do
+      it 'should be bounced back from Hybrid Handoff to How to Verify' do
```

Comment:
Note that the changes in the preconditions changes our expectations during the 50/50 state.